### PR TITLE
smoke/dust updates for the RRFS code freeze

### PR DESCRIPTION
**Pull request by haiqinli**
## DESCRIPTION OF CHANGES: 

- Update the smoke preprocessing for the new RAVE data format;
- Updates the namelist options for the updated smoke/dust code.

## TESTS CONDUCTED: 

It has been tested on Hera for the realtime forecast case.

### Machines/Platforms:

- WCOSS2
  - [ ] Cactus/Dogwood
  - [ ] Acorn
- RDHPCS
  - [X ] Hera
  - [ ] Jet
  - [ ] Orion
  - [ ] Hercules

### Test cases: 

- [ ] Engineering tests
  - [ ] Non-DA engineering test
  - [ ] DA engineering test
    - [ ] Retro
    - [ ] Ensemble
    - [ ] Parallel
- [ ] RRFS fire weather
- [X] RRFS_A:
- [X] RRFS_B:
- [ ] RTMA:
- [ ] Others:

## ISSUE: 

- Fixes the issue(s) mentioned in #9999

### CODE REVIEW

**Comment by MatthewPyle-NOAA** [comment]
@haiqinli Can you clean up the parm/FV3.input.yml conflict?  I merged a PR from @hu5970 earlier today that touched that file as well, so there are some conflicts.

**Comment by haiqinli** [comment]
@MatthewPyle-NOAA Sure, I will address the FV3.input.yml conflict.

File: scripts/HWP_tools.py
Comment (MatthewPyle-NOAA) [inline scripts/HWP_tools.py:1]: This seems like a utility script - can you convert it to being in the ush/ directory?   I may have other similar suggestions about some other python scripts, but need to see how the scripts interact.  Stay tuned.

**Comment by haiqinli** [comment]
@MatthewPyle-NOAA All the differences in FV3.input.yml are from smoke/dusts. A major update is that we will apply new diurnal cycle of smoke for 60h long forecasts (ebb_dcycle=2). The FV3.input.yml in this PR has been tested on Hera and WCOSS2, and worked well.

**Review by MatthewPyle-NOAA — COMMENTED** [review]
I need to take a closer look at the hierarchy of scripts - may need some slight reorganization to be more ready for production.

File: scripts/HWP_tools.py
Comment (haiqinli) [reply scripts/HWP_tools.py:1]: Thank you very much for looking into these scripts. There are four python scripts for smoke processing: HWP_tools.py, fire_emiss_tools.py, generate_fire_emission.py, and interp_tools.py. We will reorganize these python scripts after you check them.

**Comment by MatthewPyle-NOAA** [comment]
@haiqinli with the use of these import statements, is it important for the four python scripts to be located in the same directory?

generate_fire_emissions.py:import fire_emiss_tools as femmi_tools
generate_fire_emissions.py:import HWP_tools
generate_fire_emissions.py:import interp_tools as i_tools

Also, since the J-job is JRRFS_PROCESS_SMOKE, the production norm is to have a ${SCRIPTSdir}/exrrfs_process_smoke script (roughly matching the name of the J-job).  

So my thought would be to introduce a new ${SCRIPTSdir}/exrrfs_process_smoke.sh script that makes the call to a ${USHdir}/generate_fire_emissions.py script, with all of the python scripts imported by generate_fire_emissions.py also relocated to ${USHdir}/

Does this sound feasible?  Let me know if you need anything clarified.

**Comment by haiqinli** [comment]
@MatthewPyle-NOAA Thanks for your suggestions, and it is feasible. We will ntroduce a new ${SCRIPTSdir}/exrrfs_process_smoke.sh script that makes the call to a ${USHdir}/generate_fire_emissions.py script, with all of the python scripts imported by generate_fire_emissions.py also relocated to ${USHdir}/

**Comment by MatthewPyle-NOAA** [comment]
Thanks @haiqinli for moving those python scripts to the USHdir.  Is creating a ${SCRIPTSdir}/exrrfs_process_smoke.sh script still in your plans?  Also, parm/FV3.input.yml still needs to be cleaned up so it will merge cleanly.

**Comment by haiqinli** [comment]
@MatthewPyle-NOAA (1). Right now we don't need to create a new script of ${SCRIPTSdir}/exrrfs_process_smoke.sh.
(2). The differences in parm/FV3.input.yml are only related to smoke/dust, and will be applied for RRFS_A and RRFS_B. We have tested it with the latest ufs-weather-model, and worked well. The parm/FV3.input.yml in this PR should be good. Thanks.

**Comment by MatthewPyle-NOAA** [comment]
@BenjaminBlake-NOAA @haiqinli I would definitely like to see a ${SCRIPTSdir}/exrrfs_process_smoke.sh script be developed that makes the call to  ${USHdir}/generate_fire_emissions.py.  It should be possible to do without a lot of work by moving some of what is currently done within JRRFS_PROCESS_SMOKE into this new script.  Generally J-jobs are intended to set up the environment and call a script that does the actual processing.  Is a bit weird if you aren't used to it, but it is how jobs run in production.

**Comment by haiqinli** [comment]
@MatthewPyle-NOAA @BenjaminBlake-NOAA I agree with you. Sure, we will create a ${SCRIPTSdir}/exrrfs_process_smoke.sh script, and make a small follow-up PR soon.

**Comment by haiqinli** [comment]
@MatthewPyle-NOAA @BenjaminBlake-NOAA Thank you very much for your suggestions, ${SCRIPTSdir}/exrrfs_process_smoke.sh script has been added.

**Comment by haiqinli** [comment]
@BenjaminBlake-NOAA Thanks for pointing this out, and those lines were removed.

**Review by BenjaminBlake-NOAA — APPROVED** [review]
Thanks @haiqinli , I approve but we can wait for the final approval/confirmation from Matt.

**Comment by MatthewPyle-NOAA** [comment]
@haiqinli and @BenjaminBlake-NOAA - thanks for the updates!  I'll try to take a fresh look at it tomorrow morning.

**Pull request merged**
